test/functional/feature_asmap.py

```diff
@@ -72,6 +80,15 @@ def test_default_asmap(self):
                 self.start_node(0, [arg])
         os.remove(self.default_asmap)
 
+    def test_asmap_interaction_with_addrman_containing_entries(self):
+        self.log.info('Test bitcoind -asmap with an addrman containing new and tried entries')
+        self.fill_addrman(node_id=0)
+        self.stop_node(0)
+        shutil.copyfile(self.asmap_raw, self.default_asmap)
+        self.start_node(0, ["-asmap", "-checkaddrman=1"])
+        self.node.getnodeaddresses()  # getnodeaddresses re-runs the addrman checks
```

Comment:
nit in the last commit:

Maybe the same here with "Addrman checks started: new 2, tried 2, total 4"

Reply:
done